src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +126,75 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              { View: new WindowsHelloErrorMessageView(
+                { message: Okta.loc('enroll.windowsHello.error.notWindows', 'login') }) },
+              { selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
```

Comment:
By default subtitle is displayed above the error messages.
On the mock subtitle is placed below the error. 
https://oktainc.atlassian.net/secure/attachment/69667/07%20-%20Win%20Hello%20Challenge%20Error%20Contact%20Admin.png